profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/response_templates/poetry-confirmation-of-receiving-the-translation-request-error-configuration.tpl.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @file
+ * Template file for the POETRY mock about receiving wrong translation request.
+ *
+ * Available custom variables:
+ * - $demande_id: An array with requester data.
+ * - $message: A string with the error message.
+ */
+?>
+<POETRY xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="">
+  <request communication="asynchrone" id="<?php print implode('/', $demande_id); ?>" type="status">
+    <demandeId>
+      <codeDemandeur><?php print $demande_id['codeDemandeur']; ?></codeDemandeur>
+      <annee><?php print $demande_id['annee']; ?></annee>
+      <numero><?php print $demande_id['numero']; ?></numero>
+      <version><?php print $demande_id['version']; ?></version>
+      <partie><?php print $demande_id['partie']; ?></partie>
+      <produit><?php print $demande_id['produit']; ?></produit>
+    </demandeId>
+    <status code="-1" type="request">
+      <statusDate format="dd/mm/yyyy"><?php print date('d/m/Y'); ?></statusDate>
```

Comment:
No. 6 https://webgate.ec.europa.eu/fpfis/wikis/display/MULTISITE/Guide+into+code+reviewing+for+Drupal+7#GuideintocodereviewingforDrupal7-known_issuesQAknownissues